Installation partially fails and installs some conflicting versions

```
 ERROR: Command errored out with exit status 1:
   command: /home/guido/miniconda3/envs/windnode_abw/bin/python -u -c 'import sys, setuptools, tokenize; sys.argv[0] = '"'"'/tmp/pip-install-uqfzfs5z/pygraphviz/setup.py'"'"'; __file__='"'"'/tmp/pip-install-uqfzfs5z/pygraphviz/setup.py'"'"';f=getattr(tokenize, '"'"'open'"'"', open)(__file__);code=f.read().replace('"'"'\r\n'"'"', '"'"'\n'"'"');f.close();exec(compile(code, __file__, '"'"'exec'"'"'))' bdist_wheel -d /tmp/pip-wheel-_u1bvfp3 --python-tag cp38
       cwd: /tmp/pip-install-uqfzfs5z/pygraphviz/
  Complete output (55 lines):
  running bdist_wheel
  running build
  running build_py
  creating build
  creating build/lib.linux-x86_64-3.8
  creating build/lib.linux-x86_64-3.8/pygraphviz
  copying pygraphviz/release.py -> build/lib.linux-x86_64-3.8/pygraphviz
  copying pygraphviz/graphviz.py -> build/lib.linux-x86_64-3.8/pygraphviz
  copying pygraphviz/agraph.py -> build/lib.linux-x86_64-3.8/pygraphviz
  copying pygraphviz/__init__.py -> build/lib.linux-x86_64-3.8/pygraphviz
  copying pygraphviz/version.py -> build/lib.linux-x86_64-3.8/pygraphviz
  creating build/lib.linux-x86_64-3.8/pygraphviz/tests
  copying pygraphviz/tests/test_string.py -> build/lib.linux-x86_64-3.8/pygraphviz/tests
  copying pygraphviz/tests/test_unicode.py -> build/lib.linux-x86_64-3.8/pygraphviz/tests
  copying pygraphviz/tests/test_node_attributes.py -> build/lib.linux-x86_64-3.8/pygraphviz/tests
  copying pygraphviz/tests/test_subgraph.py -> build/lib.linux-x86_64-3.8/pygraphviz/tests
  copying pygraphviz/tests/test_edge_attributes.py -> build/lib.linux-x86_64-3.8/pygraphviz/tests
  copying pygraphviz/tests/test_attributes.py -> build/lib.linux-x86_64-3.8/pygraphviz/tests
  copying pygraphviz/tests/test_attribute_defaults.py -> build/lib.linux-x86_64-3.8/pygraphviz/tests
  copying pygraphviz/tests/test_layout.py -> build/lib.linux-x86_64-3.8/pygraphviz/tests
  copying pygraphviz/tests/test.py -> build/lib.linux-x86_64-3.8/pygraphviz/tests
  copying pygraphviz/tests/test_clear.py -> build/lib.linux-x86_64-3.8/pygraphviz/tests
  copying pygraphviz/tests/__init__.py -> build/lib.linux-x86_64-3.8/pygraphviz/tests
  copying pygraphviz/tests/test_readwrite.py -> build/lib.linux-x86_64-3.8/pygraphviz/tests
  copying pygraphviz/tests/test_setup.py -> build/lib.linux-x86_64-3.8/pygraphviz/tests
  copying pygraphviz/tests/test_html.py -> build/lib.linux-x86_64-3.8/pygraphviz/tests
  copying pygraphviz/tests/test_drawing.py -> build/lib.linux-x86_64-3.8/pygraphviz/tests
  copying pygraphviz/tests/test_graph.py -> build/lib.linux-x86_64-3.8/pygraphviz/tests
  running egg_info
  writing pygraphviz.egg-info/PKG-INFO
  writing dependency_links to pygraphviz.egg-info/dependency_links.txt
  writing top-level names to pygraphviz.egg-info/top_level.txt
  reading manifest file 'pygraphviz.egg-info/SOURCES.txt'
  reading manifest template 'MANIFEST.in'
  warning: no files found matching '*.png' under directory 'doc'
  warning: no files found matching '*.html' under directory 'doc'
  warning: no files found matching '*.txt' under directory 'doc'
  warning: no files found matching '*.css' under directory 'doc'
  warning: no previously-included files matching '*~' found anywhere in distribution
  warning: no previously-included files matching '*.pyc' found anywhere in distribution
  warning: no previously-included files matching '.svn' found anywhere in distribution
  no previously-included directories found matching 'doc/build'
  writing manifest file 'pygraphviz.egg-info/SOURCES.txt'
  copying pygraphviz/graphviz.i -> build/lib.linux-x86_64-3.8/pygraphviz
  copying pygraphviz/graphviz_wrap.c -> build/lib.linux-x86_64-3.8/pygraphviz
  running build_ext
  building 'pygraphviz._graphviz' extension
  creating build/temp.linux-x86_64-3.8
  creating build/temp.linux-x86_64-3.8/pygraphviz
  gcc -pthread -B /home/guido/miniconda3/envs/windnode_abw/compiler_compat -Wl,--sysroot=/ -Wsign-compare -DNDEBUG -g -fwrapv -O3 -Wall -Wstrict-prototypes -fPIC -I/home/guido/miniconda3/envs/windnode_abw/include/python3.8 -c pygraphviz/graphviz_wrap.c -o build/temp.linux-x86_64-3.8/pygraphviz/graphviz_wrap.o
  pygraphviz/graphviz_wrap.c:2987:10: fatal error: graphviz/cgraph.h: Datei oder Verzeichnis nicht gefunden
   #include "graphviz/cgraph.h"
            ^~~~~~~~~~~~~~~~~~~
  compilation terminated.
  error: command 'gcc' failed with exit status 1
  ----------------------------------------
  ERROR: Failed building wheel for pygraphviz
  Running setup.py clean for pygraphviz
  Building wheel for configobj (setup.py) ... done
  Created wheel for configobj: filename=configobj-5.0.6-cp38-none-any.whl size=34547 sha256=afeebb8213b531ed211f5d30103787f069be768706d00aa339117dd97a5cb561
  Stored in directory: /home/guido/.cache/pip/wheels/f1/e4/16/4981ca97c2d65106b49861e0b35e2660695be7219a2d351ee0
  Building wheel for blinker (setup.py) ... done
  Created wheel for blinker: filename=blinker-1.4-cp38-none-any.whl size=13452 sha256=b227b58a7d0f5db6092cfec707df48d4fa1765baf419635c2f180877a5e41c0a
  Stored in directory: /home/guido/.cache/pip/wheels/92/a0/00/8690a57883956a301d91cf4ec999cc0b258b01e3f548f86e89
  Building wheel for dill (setup.py) ... done
  Created wheel for dill: filename=dill-0.2.9-cp38-none-any.whl size=77404 sha256=3f0d17627534fe1f1ba6e28b6f21c20ca8bc1224adcea1bcdd2270790a6af5f6
  Stored in directory: /home/guido/.cache/pip/wheels/5b/d7/0f/e58eae695403de585269f4e4a94e0cd6ca60ec0c202936fa4a
  Building wheel for numpy (setup.py) ... done
  Created wheel for numpy: filename=numpy-1.16.5-cp38-cp38-linux_x86_64.whl size=12643399 sha256=f435a5d9398d16263d814aeaff78c24fb8474512419d4b6d9f5013bbfd30bca8
  Stored in directory: /home/guido/.cache/pip/wheels/f3/c6/40/1c3d8725494e976f620d535731122a7944523aeda91aad0d68
  Building wheel for SQLAlchemy (setup.py) ... done
  Created wheel for SQLAlchemy: filename=SQLAlchemy-1.3.11-cp38-cp38-linux_x86_64.whl size=1216850 sha256=27d8087f7ecc20a48a62a7adc8d6a5adfee157cb666608f28ce0f14be3b97a5d
  Stored in directory: /home/guido/.cache/pip/wheels/a3/67/7d/6c41104a1a08ff1a25e260d3edec3ac19203141d1aaa2f0975
Successfully built oemof shapely matplotlib egoio configobj blinker dill numpy SQLAlchemy
Failed to build pygraphviz
ERROR: oemof 0.3.1 has requirement networkx<2.4, but you'll have networkx 2.4 which is incompatible.
ERROR: oemof 0.3.1 has requirement pandas<0.25,>=0.18.0, but you'll have pandas 0.25.3 which is incompatible.
Installing collected packages: blinker, dill, numpy, pytz, six, python-dateutil, pandas, nose, PyUtilib, ply, appdirs, pyomo, decorator, networkx, oemof, shapely, SQLAlchemy, GeoAlchemy2, cycler, kiwisolver, pyparsing, matplotlib, psycopg2-binary, entrypoints, jeepney, pycparser, cffi, cryptography, secretstorage, keyring, keyrings.alt, chardet, idna, urllib3, requests, oedialect, egoio, pyproj, pygraphviz, configobj, windnode-abw
    Running setup.py install for pygraphviz ... error
    ERROR: Command errored out with exit status 1:
     command: /home/guido/miniconda3/envs/windnode_abw/bin/python -u -c 'import sys, setuptools, tokenize; sys.argv[0] = '"'"'/tmp/pip-install-uqfzfs5z/pygraphviz/setup.py'"'"'; __file__='"'"'/tmp/pip-install-uqfzfs5z/pygraphviz/setup.py'"'"';f=getattr(tokenize, '"'"'open'"'"', open)(__file__);code=f.read().replace('"'"'\r\n'"'"', '"'"'\n'"'"');f.close();exec(compile(code, __file__, '"'"'exec'"'"'))' install --record /tmp/pip-record-obrkrcps/install-record.txt --single-version-externally-managed --compile
         cwd: /tmp/pip-install-uqfzfs5z/pygraphviz/
    Complete output (32 lines):
    running install
    Trying dpkg
    Trying pkg-config
    Package libcgraph was not found in the pkg-config search path.
    Perhaps you should add the directory containing `libcgraph.pc'
    to the PKG_CONFIG_PATH environment variable
    No package 'libcgraph' found
    Traceback (most recent call last):
      File "<string>", line 1, in <module>
      File "/tmp/pip-install-uqfzfs5z/pygraphviz/setup.py", line 70, in <module>
        setup(
      File "/home/guido/miniconda3/envs/windnode_abw/lib/python3.8/site-packages/setuptools/__init__.py", line 145, in setup
        return distutils.core.setup(**attrs)
      File "/home/guido/miniconda3/envs/windnode_abw/lib/python3.8/distutils/core.py", line 148, in setup
        dist.run_commands()
      File "/home/guido/miniconda3/envs/windnode_abw/lib/python3.8/distutils/dist.py", line 966, in run_commands
        self.run_command(cmd)
      File "/home/guido/miniconda3/envs/windnode_abw/lib/python3.8/distutils/dist.py", line 985, in run_command
        cmd_obj.run()
      File "/tmp/pip-install-uqfzfs5z/pygraphviz/setup_commands.py", line 44, in modified_run
        self.include_path, self.library_path = get_graphviz_dirs()
      File "/tmp/pip-install-uqfzfs5z/pygraphviz/setup_extra.py", line 162, in get_graphviz_dirs
        include_dirs, library_dirs = _try_configure(include_dirs, library_dirs, _pkg_config)
      File "/tmp/pip-install-uqfzfs5z/pygraphviz/setup_extra.py", line 117, in _try_configure
        i, l = try_function()
      File "/tmp/pip-install-uqfzfs5z/pygraphviz/setup_extra.py", line 72, in _pkg_config
        output = S.check_output(['pkg-config', '--libs-only-L', 'libcgraph'])
      File "/home/guido/miniconda3/envs/windnode_abw/lib/python3.8/subprocess.py", line 411, in check_output
        return run(*popenargs, stdout=PIPE, timeout=timeout, check=True,
      File "/home/guido/miniconda3/envs/windnode_abw/lib/python3.8/subprocess.py", line 512, in run
        raise CalledProcessError(retcode, process.args,
    subprocess.CalledProcessError: Command '['pkg-config', '--libs-only-L', 'libcgraph']' returned non-zero exit status 1.
    ----------------------------------------
ERROR: Command errored out with exit status 1: /home/guido/miniconda3/envs/windnode_abw/bin/python -u -c 'import sys, setuptools, tokenize; sys.argv[0] = '"'"'/tmp/pip-install-uqfzfs5z/pygraphviz/setup.py'"'"'; __file__='"'"'/tmp/pip-install-uqfzfs5z/pygraphviz/setup.py'"'"';f=getattr(tokenize, '"'"'open'"'"', open)(__file__);code=f.read().replace('"'"'\r\n'"'"', '"'"'\n'"'"');f.close();exec(compile(code, __file__, '"'"'exec'"'"'))' install --record /tmp/pip-record-obrkrcps/install-record.txt --single-version-externally-managed --compile Check the logs for full command output.
```

--- a/README.md
+++ b/README.md
@@ -21,6 +21,24 @@ To run the model, you also need a solver to be installed such as CBC or Gurobi.
 On Linux, you can install CBC with `apt install coinor-cbc`. Make sure the
 solver is set in the run configuration dict in `run_scenario.py`.
 
+### Trouble shooting
+
+#### Installation of `psutils`
+
+During the installtion of `windnode_abw`, the package `psutils` might fail. Install the system package `python3-dev` with
+
+```bash
+sudo apt install python3-dev
+```
+
+#### Installation of `pygraphviz`
+
+During the installtion of `windnode_abw`, the package `pygraphviz` might fail. Install additional system packages with
+
+```bash
+sudo apt install python3-dev graphviz libgraphviz-dev pkg-config
+```
+
 ### Setup postgres database with docker (optional)
 
 **Note** You don't have to necessarily use docker to create a Postgres database. Using a native installtion works as well

--- a/docs/usage_notes.rst
+++ b/docs/usage_notes.rst
@@ -27,6 +27,29 @@ Install requirements manually in your venv, **egoio** should be the last. Instal
 To run the model, you also need a solver to be installed such as CBC or Gurobi. On Linux, you can install CBC with
 `apt install coinor-cbc`. Make sure the solver is set in the run configuration dict in `run_scenario.py`.
 
+Trouble shooting
+^^^^^^^^^^^^^^^^
+
+Installation of ``psutils``
+"""""""""""""""""""""""""""
+
+During the installtion of ``windnode_abw``, the package ``psutils``
+might fail. Install the system package ``python3-dev`` with
+
+.. code:: bash
+
+   sudo apt install python3-dev
+
+Installation of ``pygraphviz``
+""""""""""""""""""""""""""""""
+
+During the installtion of ``windnode_abw``, the package ``pygraphviz``
+might fail. Install additional system packages with
+
+.. code:: bash
+
+   sudo apt install python3-dev graphviz libgraphviz-dev pkg-config
+
 Setup postgres database with docker (optional)
 ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
 

